.coafile

```diff
@@ -39,7 +39,7 @@ max_lines_per_file = 1000
 
 [rst]
 bear = reSTCheckBear
-files = README.rst
+files = **/*.rst
```

Comment:
to *only* collect subdirs, `*/**.rst` is correct.

But... I cant reproduce this... but I can raise a bug: https://github.com/coala/coala/issues/3369